.exports

```diff
@@ -17,13 +17,9 @@ if [[ -x /opt/homebrew/bin/brew ]]; then
     # See https://docs.brew.sh/Shell-Completion#configuring-completions-in-zsh
     FPATH="$HOMEBREW_PREFIX/share/zsh/site-functions:$FPATH"
 
-    # Credit: https://superuser.com/a/1092328/144662
-    # Credit: https://thevaluable.dev/zsh-completion-guide-examples/
-    autoload -Uz compinit
-    compinit
-
-    source `brew --prefix`/share/zsh-autosuggestions/zsh-autosuggestions.zsh
-    source `brew --prefix`/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
+    if [[ -d `brew --prefix`/share/zsh-syntax-highlighting ]]; then
+        source `brew --prefix`/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
+    fi
```

Comment:
Here I have 2 choices, clone this plugin to `$ZSH_CUSTOM` directory or install via `homebrew` and in this case I choose to use brew instead.